pmm-app/src/pmm-qan/panel/components/Filters/Filters.constants.ts

```diff
@@ -10,10 +14,34 @@ export const FILTERS_GROUPS = [
   {
     name: 'Cluster',
     dataKey: 'cluster',
+    getDashboardURL: (value: string) => {
+      const serviceType = getServiceType(value, 'cluster');
+      let dashboardURL = '';
+
+      if (serviceType === 'mysql') {
```

Comment:
Would be better if use serviceType from enum

Reply:
Should I use existing enum in Details or create a new one here in Filters folder?

Reply:
Done!